src/lib/components/AudioPlayer/audio-player-state.ts

```diff
@@ -21,15 +24,18 @@ export interface AudioFileInfo {
 // Under the hood this will keep track of each of the files to be included in the clip sequence, including their
 // HTMLAudioElement instances.
 class _MultiClipAudioState {
+    globalId: string;
     currentClipIndex: number;
     clipSequence: _AudioClip[];
     syncSeekPositionTimer: ReturnType<typeof setInterval> | null;
     seekCooldown = 0;
     _rangeValue: number;
     _timeDisplay: string;
     _totalTimeDisplay: string;
+    unsubscribePlayingGlobalId: Unsubscriber | undefined;
 
     constructor(files: AudioFileInfo[]) {
+        this.globalId = Math.random().toString(36).substring(2, 15);
```

Comment:
Not exactly sure the use case, but would a GUID make sense here? In the content manager I added that uuid library, like `import { v4 as uuid } from 'uuid';` and then can just call `uuid()` to get one.